package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@superhuman/react-frame-component",
-  "version": "2.0.0",
```

Comment:
What happened to 2.0.1? : P

Reply:
I don't know - someone already published it